.infrastructure/todoapp-pod.yml

```diff
@@ -21,3 +30,37 @@ spec:
         port: 8080
       initialDelaySeconds: 5
       periodSeconds: 5
+
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: todoapp-2
+  namespace: todoapp
+  labels:
+    app: todolist
+spec:
+  containers:
+  - name: todoapp
+    image: ikulyk404/todoapp:3.0.0
+    ports:
+    - containerPort: 8080
+    env:
+    - name: ENVIRONMENT
+      value: "production"
+    - name: DEBUG
+      value: "false"
+    - name: API_URL
+      value: "http://todoapp-clusterip"
```

Comment:
What are these variables needed for?

Reply:
variable for Django debugging https://docs.djangoproject.com/en/5.1/ref/settings/#debug
if I correctly undestood task (task 6 Set all env values for the container from pod’s manifest)
https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/

Reply:
The variable that needs to be added is `PYTHONUNBUFFERED`. Adding `DEBUG` doesn't make sense since its default value is already false. Also, I don't see the need for `API_URL`